### PR TITLE
Editorial nits based on shepherd review

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-mmusic-t140-usage-data-channel.xml [hunk 1]
```diff
@@ -279,32 +279,33 @@
...
+                explicitly mark the data channel) or recvonly, it MUST be prepared to receive
+                T.140 data as soon as the state of the T.140 data channel allows it. 
               </t>
           </section> 
           <section anchor="sec.sdp.dcsa.dir.neg.ans" title="Generating an Answer">
               <t>
                 When the answerer accepts an offer, and marks the direction of the text 
-                in the corresponding answer, the marking is based on the marking (explicit or implicit) 
-                in the offer.
+                in the corresponding answer, the marking is based on the marking (or the lack
```
Comment (fandreas) [inline draft-ietf-mmusic-t140-usage-data-channel.xml]: Suggest changing "marking is based on the marking" to "direction is based on the marking"

File: draft-ietf-mmusic-t140-usage-data-channel.xml [hunk 2]
```diff
@@ -350,8 +350,9 @@
             for a T.140 data channel offering real-time text
             conversation in Spanish and Esperanto, and an m= section
             in the associated answer accepting Esperanto. The maximum
-            character transmission rate is set to 20 and the default
-            text transmission direction "sendrecv" applies.
+            character transmission rate is set to 20. As the offerer and
+            answerer has not explicitly indicated the real-time text
```
Comment (fandreas) [inline draft-ietf-mmusic-t140-usage-data-channel.xml]: s/has/have

File: draft-ietf-mmusic-t140-usage-data-channel.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-mmusic-t140-usage-data-channel.xml]: Will fix as suggested.

File: draft-ietf-mmusic-t140-usage-data-channel.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-mmusic-t140-usage-data-channel.xml]: Will fix.

**Review by gunnarhm — APPROVED** [review]
Look good.

File: draft-ietf-mmusic-t140-usage-data-channel.xml [hunk 3]
```diff
@@ -70,7 +70,7 @@
         see Section 3.2 of <xref target="I-D.ietf-rtcweb-data-channel"/>.
       </t>
       <t>
-        The brief notation "T.140" is used as a synonym for the text
+        The brief notation "T.140" is used as an acronym for the text
```
Comment (gunnarhm) [inline draft-ietf-mmusic-t140-usage-data-channel.xml]: Acronym should be a name composed by the initial letters of a sequence of words. How about just "a name" instead?

**Review by gunnarhm — COMMENTED** [review]
All fine except "acronym" on line 73. See specific comment proposing "a name" instead.

File: draft-ietf-mmusic-t140-usage-data-channel.xml (hunk 3 above)
Comment (cdh4u) [reply draft-ietf-mmusic-t140-usage-data-channel.xml]: I can use "a name". I am sure the RFC editor will let us know if he/she thinks there is something better.

**Pull request merged**